cmd/init.go

```diff
@@ -80,3 +87,83 @@ func initStatsClient() {
 
 	log.AddHook(hooks.NewLogrusHook(statsClient, globalConfig.Stats.ErrorsSection))
 }
+
+func initStatsExporter() {
+	var err error
+	logger := log.WithField("stats.exporter", globalConfig.Stats.Exporter)
+
+	// Register stats exporter according to config
+	switch globalConfig.Stats.Exporter {
+	case "datadog":
+	case "stackdriver":
+		logger.Warn("Not implemented!")
+		return
+	case "prometheus":
+		err = initPrometheusExporter()
+		break
+	default:
+		logger.Info("Unsupported or invalid stats exporter was specified")
+		return
+	}
+
+	if err != nil {
+		logger.Warn("Failed initialising stats exporter")
+		return
+	}
+
+	// Configure/Register stats views
+	view.SetReportingPeriod(time.Second)
+
+	vv := append(ochttp.DefaultServerViews, obs.AllViews...)
+
+	if err := view.Register(vv...); err != nil {
+		log.WithError(err).Warn("Failed to register server views")
+	}
+}
+
+func initPrometheusExporter() (err error) {
+	obs.PromExporter, err = prometheus.NewExporter(prometheus.Options{
+		Namespace: "janus",
```

Comment:
namespace can be configurable IMO, with `janus` as default value

Reply:
Yes! I believe we have `serviceName` somewhere in the config which can be used here.